echopype/convert/set_groups_ek60.py

```diff
@@ -405,46 +405,34 @@ def set_beam(self) -> xr.Dataset:
                     beam_params["beam_type"],
                     {"long_name": "type of transducer (0-single, 1-split)"},
                 ),
-                # TODO: check EK60 data spec:
-                #  the beamwidths provided are most likely 2-way beamwidth so below needs to change
-                "beamwidth_receive_alongship": (
+                "beamwidth_twoway_alongship": (
                     ["channel"],
                     beam_params["beamwidth_alongship"],
                     {
                         "long_name": "Half power one-way receive beam width along "
                         "alongship axis of beam",
                         "units": "arc_degree",
                         "valid_range": (0.0, 360.0),
+                        "comment": (
+                            "Introduced in echopype for Simrad echosounders to avoid potential confusion with convention definitions. "  # noqa
+                            + "The alongship angle corresponds to the minor angle in SONAR-netCDF4 vers 2. "  # noqa
```

Comment:
The + sign is unnecessary for string concatenation in this context. I'm totally fine with leaving it there for now, though I think the strings are a bit more readable w/o it (and I believe we typically don't use +'s in such multi-line strings in echopype).

This comments applies to all other `comment` multi-line strings in this PR.

Reply:
Also, should we include the (roughly) corresponding variable(s) from v1? ie, `beamwidth_receive_minor` and `beamwidth_transmit_minor`. eg, replace ""The convention defines one-way transmit or receive beamwidth, but Simrad echosounders record two-way beamwidth in the data." with ""The convention defines one-way transmit or receive beamwidth (beamwidth_receive_minor and beamwidth_transmit_minor), but Simrad echosounders record two-way beamwidth in the data."

This comment applies to other, related changse in this PR.

Reply:
I replaced the original one with the new entry `"The convention defines one-way transmit or receive beamwidth (beamwidth_receive_minor and beamwidth_transmit_minor), but Simrad echosounders record two-way beamwidth in the data."`